cita-executor/evm/Cargo.toml

```diff
@@ -16,6 +16,9 @@ rlp = { git = "https://github.com/cryptape/cita-common.git", branch = "develop"
 rustc-hex = "1.0"
 db = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
 
+cita-vm = { git = "https://github.com/cryptape/cita-vm", branch = "cita" }
```

Comment:
Do not depend cita-vm in evm. The folder `evm` will be deleted very soon. So please separate the file `storage.rs` out of this folder.

Reply:
Currently, I am not clear enough about the directoy structure in cita-executor after you refactored.
I plan to find an appropriate path for `storage.rs`